src/map/entities/baseentity.h

```diff
@@ -284,6 +284,8 @@ class CBaseEntity
     uint32 GetLocalVar(std::string var);
     void   SetLocalVar(std::string var, uint32 val);
 
+    std::map<std::string, uint32>& GetLocalVars();
```

Comment:
auto GetLocalVars() -> std::map<std::string, uint32>&;

Reply:
> auto GetLocalVars() -> std::map<std::string, uint32>&;

Thanks, I updated the PR!